build-system-tests/templates/components/react/vite/template-tsconfig-vite-latest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "target": "ESNext",
+    "lib": [
+      "DOM",
+      "DOM.Iterable",
+      "ESNext"
+    ],
+    "module": "ESNext",
+    "skipLibCheck": true,
+    /* Bundler mode */
```

Comment:
Should we remove the comment assuming this is a pure json file?

Reply:
Yes, I thought about that too and kept it there because it's generated by Vite CLI... I'm open to either way because I'd like to have a follow up to modify this file at run time (to add `allowJS: true`) instead of using a template.

Do you still prefer removing it?